src/intrinsics.cpp

```diff
@@ -986,6 +986,20 @@ static Value *emit_intrinsic(intrinsic f, jl_value_t **args, size_t nargs,
                                    ArrayRef<Type*>(x->getType())),
          FP(x), FP(y), FP(z));
     }
+    HANDLE(muladd_float,3)
+#ifdef LLVM34
```

Comment:
`LLVM34` doesn't do what you probably think it does -- it means "LLVM 3.4 or later".